documentation/how-to-topics/enable-windows-authentication.md

```diff
@@ -0,0 +1,127 @@
+# How to Enable Windwos Authentication on OpenSilver App
+
+To enable windows authentication on Opensilver Application update `"windowsAuthentication": true,` and `"anonymousAuthentication": false,` under `iisSettings` section in `launchSettings.json`.
+
+Update `launchSettings.json` with the following content:
+```
+{
+  "iisSettings": {
+    "windowsAuthentication": true,
+    "anonymousAuthentication": false,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:55591/",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "WindowsAuthSample.Browser": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:55592/"
+    }
+  }
+}
+```
+### IIS Settings
+
+Double click on Authentication in IIS.
+
+<img src="/images/how-to-topics/windows-auth1.png" alt="windows" title="IIS-settings.png" style="border: 2px solid #555;" /><br />
+
+Make sure only "Basic authentication" is enabled.
+
+<img src="/images/how-to-topics/windows-auth2.png" alt="windows" title="IIS-settings.png" style="border: 2px solid #555;" /><br />
+
+### Access UserName
+
+Add following package to project:
```

Comment:
We have several packages, so let's say "Add the following packages to the project:"